common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/OSGI-INF/l10n/bundle.properties

```diff
@@ -0,0 +1,18 @@
+ ###############################################################################
+ # Copyright (c) 2016 Thales Global Services S.A.S.

```

Comment:
should this copyright year be updated? there are changes but the project name is changed !!